frontend/assets/svg/partners/Crucial.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+function SvgMicron(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
Were these auto-generated? Would it be worth not committing them and just generating them from the SVGs on each installation (or something like that)?

Reply:
No I converted these by hand, though I'm sure we could auto generate the components from the files if we wanted to.

@federicobadini did you autogenerate the SVG components in #918? And if you did, does @andyg0808's suggestion make sense?

Reply:
Upon further inspection I discovered we have an npm script `transform-svg` in `frontend/` that seems to generate these components from the svg files.

Looking into transforming them at build time though...

Reply:
Sorry for the late reply but I missed the notification. 
You figured it out correctly: we used that npm script to do the transformation